hathor/manager.py

```diff
@@ -423,14 +423,15 @@ def validate_new_tx(self, tx: BaseTransaction) -> bool:
             tx = cast(Block, tx)
             assert tx.hash is not None  # XXX: it appears that after casting this assert "casting" is lost
 
-            # Validate minimum block difficulty
-            block_weight = self.calculate_block_difficulty(tx)
-            if tx.weight < block_weight - settings.WEIGHT_TOL:
-                raise InvalidNewTransaction(
-                    'Invalid new block {}: weight ({}) is smaller than the minimum weight ({})'.format(
-                        tx.hash.hex(), tx.weight, block_weight
+            if self.state != self.NodeState.INITIALIZING:
```

Comment:
What if we check one every 120 blocks, i.e., once per hour? To remove this logic from here, we can add a `skip_block_weight_verification` parameter to this method.

Reply:
Can you check this commit and see if it's good? https://github.com/HathorNetwork/hathor-core/pull/124/commits/e83a2052ca868ac932b848d93ffd3a6e5435912d

I would just discuss about this number, I added a config on settings with this and first I set as 1000. I am not sure the benefit of having 6k verifications instead of 700. But it's easy to change

Reply:
I think it is great. Thanks!